src/main/java/com/cburch/logisim/std/io/DotMatrixHDLGeneratorFactory.java

```diff
@@ -61,115 +59,33 @@
-      if (HDL.isVHDL()) {
-        var indent = "   ";
-        if (!oneRow) {
-          contents.add(indent + label + "_0 : FOR r IN " + (rows - 1) + " DOWNTO 0 GENERATE");
-          indent  += "   ";
-          rowName  += "(r)";
-        }
-        if (!oneCol) {
-          contents.add(indent + label + "_1 : FOR c IN " + (cols - 1) + " DOWNTO 0 GENERATE");
-          indent += "   ";
-          colName += "(c)";
-        }
-
-        var content = indent + HDLGeneratorFactory.LocalOutputBubbleBusname + "(";
-
-        content += ((oneRow) ? "" : "r" + ((oneCol) ? "" : "*" + cols + "+"))
-                    + ((oneCol) ? "" : "c")
-                    + ((oneRow && oneCol) ? "" : "+");
-
-        content += componentInfo.GetLocalBubbleOutputStartId() + ") <= " + rowName + " AND " + colName + ";";
-        contents.add(content);
-
-        if (!oneCol) {
-          indent = indent.substring(0, indent.length() - 3);
-          contents.add(indent + "END GENERATE " + label + "_1;");
-        }
-        if (!oneRow) {
-          indent = indent.substring(0, indent.length() - 3);
-          contents.add(indent + "END GENERATE " + label + "_0;");
-        }
-
-      } else {
-        var indent = "   ";
-        if (!oneRow || !oneCol) {
-          contents.add(indent + "genvar " + ((oneRow) ? "" : label + "_r" + ((oneCol) ? "" : ",")) + ((oneCol) ? "" : label + "_c") + ";");
-          contents.add(indent + "generate");
-          indent += "   ";
-        }
-        if (!oneRow) {
-          contents.add(indent + "for (" + label + "_r = 0 ; " + label + "_r < " + rows + "; " + label + "_r=" + label + "_r+1)");
-          contents.add(indent + "begin:" + label + "_0");
-          indent += "   ";
-          rowName += "[" + label + "_r]";
-        }
-        if (!oneCol) {
-          contents.add(indent + "for (" + label + "_c = 0 ; " + label + "_c < " + cols + "; " + label + "_c=" + label + "_c+1)");
-          contents.add(indent + "begin:" + label + "_1");
-          indent += "   ";
-          colName += "[" + label + "_c]";
-        }
-
-        var content = indent + HDL.assignPreamble() + HDLGeneratorFactory.LocalOutputBubbleBusname + "[";
-
-        content += ((oneRow) ? "" : label + "_r" + ((oneCol) ? "" : "*" + cols + "+"))
-                    + ((oneCol) ? "" : label + "_c")
-                    + ((oneRow && oneCol) ? "" : "+");
-
-        content += componentInfo.GetLocalBubbleOutputStartId() + "] = " + rowName + " & " + colName + ";";
-        contents.add(content);
-
-        if (!oneCol) {
-          indent = indent.substring(0, indent.length() - 3);
-          contents.add(indent + "end");
-        }
-        if (!oneRow) {
-          indent = indent.substring(0, indent.length() - 3);
-          contents.add(indent + "end");
-        }
-
-        if (!oneRow || !oneCol) {
-          indent = indent.substring(0, indent.length() - 3);
-          contents.add(indent + "endgenerate");
+      for (var r = 0; r < rows; r++) {
```

Comment:
No single letter vars please. Make it `row`, `singleRow` etc. Same for `c` below.